src/test/java/uk/gov/digital/ho/hocs/workflow/api/dto/MigrateCaseResponseTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.digital.ho.hocs.workflow.api.dto;
+
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MigrateCaseResponseTest {
+
+    @Test
+    public void getMigrateCaseResponse() {
+
+        UUID caseUUID = UUID.randomUUID();
+        String reference = "someRef";
+
+        MigrateCaseResponse migrateCaseResponse = new MigrateCaseResponse(caseUUID, reference);
+
+        assertThat(migrateCaseResponse.getUuid()).isEqualTo(caseUUID);
+        assertThat(migrateCaseResponse.getReference()).isEqualTo(reference);
+
+    }
+
+    @Test
+    public void getMigrateCaseResponseNull() {
+
+        MigrateCaseResponse migrateCaseResponse = new MigrateCaseResponse(null, null);
```

Comment:
If you're creating a request to migrate a case, and your input is `null`, `null`, would you be happen with a positive response? Which would, presumably continue to try to migrate case with ID `null`?